doc/doctrine/uploadable.md

```diff
@@ -78,7 +78,7 @@ class Document
  public function buildForm(FormBuilderInterface $builder, array $options)
  {
       $builder
-          ->add('file', 'file', array(
+          ->add('file', \Symfony\Component\Form\Extension\Core\Type\FileType::class, array(
```

Comment:
Yep ! But, it's better with short notation  ``FileType::class`` and with ``use Symfony\Component\Form\Extension\Core\Type\FileType;`` in header, no?